lib/mayaUsd/render/MaterialXGenOgsXml/Nodes/TexcoordNodeMaya.cpp

```diff
@@ -5,10 +5,47 @@
 
 #include "TexcoordNodeMaya.h"
 
+#include <mayaUsd/render/MaterialXGenOgsXml/CombinedMaterialXVersion.h>
+#include <mayaUsd/render/MaterialXGenOgsXml/OgsXmlGenerator.h>
+
 #include <MaterialXGenShader/Shader.h>
 
 MATERIALX_NAMESPACE_BEGIN
 
+namespace {
+std::string geomNameFromIndex(const std::string& index)
+{
+    std::string geomname = OgsXmlGenerator::getPrimaryUVSetName();
+
+    // The code below which is trying to handle non-zero indices has little chance of working. Main
+    // reason is that our main client (USD) will only handle UV0 by adding an extra primvar in the
+    // set of required primvars. Since this is the set that will be used by the mesh hydra scene
+    // delegate, the primvars for non-zero index will end up missing in the cached geometry data and
+    // prevent copying all necessary buffers to render the material correctly.
+
+    // Fixing this would require extending USD's GetAdditionalPrimvarProperties() to return a
+    // functor instead of a hardcoded Token. This way we could add the equivalent of this code as a
+    // functor in the MaterialX parser, and the _ExtractPrimvarsFromNode function found in
+    // pxr\usdImaging\usdImaging\materialParamUtils.cpp would then be able to handle indexed values.
+#if 0
```

Comment:
The old code used to try to use the index. With this #if 0, index is not considered at all. Doesn't this create a regression?